RFCs/Image Filter.md

```diff
@@ -0,0 +1,156 @@
+### Context:
+
+When kpack produces a build, certain attributes are surfaced to the end-user that serve to provide additional context about the build and why it was initiated.   Some of this metadata intends to provide end-users with a bill of materials about the build.  Such fields include `Image`, `Pod Name`, `Builder`, `Run Image`, `Source`, `Url`, and `Revision`.  
+
+We provide additional fields in the kpack build object that are geared towards reassuring the user that a successful build occurred as well as why the build was initiated. Because of their usefulness in conveying this information to the user, the build’s `STATUS` and `REASON` attributes are surfaced prominently in the `kp build list` table.  
+
+### Problem:
+
+Some users of the kpack CLI have complained that it is difficult to get a broad view of kpack images and builds at scale at a glance without needing to further drill down to discover more details about images or builds. This problem led to specific feature requests from Jason Morgan (VMware Solutions Engineer) to unlink the build list table from images and enable users to filter builds by their `REASON` and `STATUS` regardless of their parent image.   
+
+### Use Cases:
+
+Based on evidence provided by Jason and conducting user interviews with VMware field early-adopters, I perceive the following workflows to pose more complexity than is acceptable when a user is managing many images using the kpack CLI.
+
+* A dev ops user updates a stack that is used by most if not all applications built by kpack and wants to validate that the images are in a ready state
+* A dev ops user updates a store with new buildpacks and wants to validate that the affected images rebuilt successfully
+* A dev ops user checking in to make sure that most Images are in a ready state so they can be sure that kpack is being used properly 
+
+### Outcome:
+
+Propose an additional feature or set of features that enable users to query kpack images based on their `STATUS` and `REASON` such that it is easier for users to perform the workflows in the “Use Cases” section.
+
+Although the feature request stated in the “Problem” section is to query builds, it is probably more productive to query images because end-users most likely care about the current state of their application build and not builds that happened in the past.  Additionally, querying based on builds would pose a technical problem as the CLI would need to present builds and their build numbers outside the context of their associated image.  
+
+### Prior Art:
```

Comment:
[Field Selectors](https://kubernetes.io/docs/concepts/overview/working-with-objects/field-selectors/) would probably be relevant prior art.